vsts/pipelines/templates/_signBinary.yml

```diff
@@ -41,7 +41,12 @@ steps:
     dotnet publish -r linux-x64 -c Release src\BuildScriptGeneratorCLI\BuildScriptGeneratorCli.csproj
   displayName: 'dotnet publish and after setting git_commit and build_number as env variable'
   condition: and(succeeded(), eq(variables['setSignTypeVariable.SignType'], 'real'))
-  
+
+- task: UseDotNet@2
+  displayName: 'Use .NET Core sdk 3.1.x'
```

Comment:
this appears a bit late step for this task (before line 37 or 31 would be better)

Reply:
good catch .. we need to do it before dotnet publish